src/os/bluestore/KernelDevice.cc

```diff
@@ -392,39 +396,27 @@ void KernelDevice::aio_submit(IOContext *ioc)
   ioc->num_pending -= pending;
   assert(ioc->num_pending.load() == 0);  // we should be only thread doing this
 
-  bool done = false;
-  while (!done) {
-    FS::aio_t& aio = *p;
+  for (auto it = p; it != e; it++) {
```

Comment:
@aclamk How about using range-based for loop here, that's my opinion..
Your thoughts?